### PR TITLE
Made sort posts

### PR DESCRIPTION
**Pull request by mikeshukshin**

### CODE REVIEW

File: desktop.blocks/forum-sidebar/forum-sidebar.bemtree [hunk 1]
```diff
@@ -2,6 +2,22 @@ block('forum-sidebar')(
     content()(function() {
         return [
             {
+                block: 'form-sorting',
```
Comment (tadatuta) [inline desktop.blocks/forum-sidebar/forum-sidebar.bemtree]: why not just to mix `form-sorting` to `select`?

File: desktop.blocks/forum-sorting/forum-sorting.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+modules.define(
+    'forum-sidebar',
+    ['i-bem__dom', 'jquery', 'location'],
+    function(provide, BEMDOM, $, location) {
+
+    provide(BEMDOM.decl(this.name, {
+        onSetMod: {
+            js: {
+                inited: function() {
+                    this._select = this.findBlockInside('select');
```
Comment (tadatuta) [inline desktop.blocks/forum-sorting/forum-sorting.js:10]:
``` js
BEMDOM.select.on(this.domElem, 'change', this._onSelectChange); // as you don't need current context actually
```

Comment (mikeshukshin) [inline desktop.blocks/forum-sorting/forum-sorting.js:10]: yes, context is unnecessary. I forgot delete

Comment (tavriaforever) [inline desktop.blocks/forum-sorting/forum-sorting.js:10]: after change - we get value

**Pull request closed without merging**